processing/README.md

```diff
@@ -40,29 +40,29 @@ Run in development mode with auto-reloading
 uvicorn app.main:app --host 0.0.0.0 --port 8000 --reload
 ```
 
-## Start with docker
+## Run with docker compose with ollama profile
 
-### Build Image
-
-This will create image called `processing`
+Run (append `-d` for deattached mode)
 
 ```bash
-docker build -t processing .
+docker compose --profile ollama up
```

Comment:
please add information that `docker compose up` will run everything except ollama, and `docker compose -profile "*"  up` will run everything